ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -161,6 +191,54 @@
                         default(crc_ci_bootstrap_networks_out[compute_node].default.ip) |
                         ansible.utils.ipaddr('address')
                       }}
+
```

Comment:
we need to initialize ansiblevars as a dict here, so kustomize can add more elements:
```
    - op: add
      path: spec/nodes/edpm-{{ compute_node }}/ansible/ansibleVars
      value: {}
```
This is only needed here for /spec/nodes since /spec/nodeTemplate already have that dict.

Reply:
Thx. Changed. Lets see if that will help :)